wear/src/main/java/chat/rocket/android/account/ui/AccountFragment.kt

```diff
@@ -0,0 +1,85 @@
+package chat.rocket.android.account.ui
+
+import android.app.Fragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import chat.rocket.android.R
+import chat.rocket.android.account.presentation.AccountPresenter
+import chat.rocket.android.account.presentation.AccountView
+import chat.rocket.android.util.DrawableHelper
+import chat.rocket.android.util.showToast
+import chat.rocket.android.util.ui
+import chat.rocket.common.model.UserStatus
+import dagger.android.AndroidInjection
+import kotlinx.android.synthetic.main.fragment_account.*
+import javax.inject.Inject
+
+class AccountFragment : Fragment(), AccountView {
+
+    @Inject
+    lateinit var presenter: AccountPresenter
+
+    companion object {
```

Comment:
Just a recommendation: Put `companion object` at the bottom of a class (to follow our coding style).